app/dashboard/forms.py

```diff
@@ -70,11 +70,38 @@ def __init__(self, *args, **kwargs):
             self.fields["y"].choices = tuple(choices)
 
 
+class StackedTimeseriesGraphForm(forms.Form):
+    energy_vector = forms.MultipleChoiceField(
```

Comment:
```suggestion
    energy_vector = forms.ChoiceField(
```
According to what was said in the working day, there should be only one energy vector for stacked graphs